sql/moz-fx-data-shared-prod/telemetry_derived/desktop_eng_client_v1/query.sql

```diff
@@ -0,0 +1,28 @@
+SELECT
```

Comment:
we need to add channel (client level and aggregation)

Reply:
Sorry this is probably a beginner question, but where do I pull that from to add it? I don't see it on active_users - is it somewhere else?

Reply:
the one from `clients_daily_v6` works for me

Reply:
@kwindau let's use `normalized_channel` (and call it normalized_channel as well)

Reply:
Will do, thanks!